src/main/java/com/adyen/model/checkout/InstallmentOption.java

```diff
@@ -0,0 +1,213 @@
+/*
```

Comment:
very minor: could you please add the correct copyright and auto-indent the file from intellij?

Reply:
will do it, thanks.